components/clp-py-utils/clp_py_utils/clp_config.py

```
@@ -108,23 +111,59 @@
 Port = Annotated[int, Field(gt=0, lt=2**16)]
 ZstdCompressionLevel = Annotated[int, Field(ge=1, le=19)]
 
+StrEnumSerializer = PlainSerializer(serialize_enum)
+
+PathStr = Annotated[pathlib.Path, PlainSerializer(serialize_path)]
+
+
+def _validate_directory(value: Any):
```

Ruff (0.14.0)

Missing return type annotation for private function `_validate_directory`

(ANN202)